src/widgets/Menu.js

```diff
@@ -39,12 +39,23 @@ export default class Menu extends createjs.Container {
       role: AccessibilityModule.ROLES.MENUITEM,
     });
     this.addChild(this._label);
-    // todo: underline the first letter that corrsponds to the accessKey
 
-    const bounds = this._label.getBounds();
+    const {
+      x, y, width, height,
+    } = this._label.getBounds();
+    const underLine = new createjs.Shape();
+    underLine.graphics.setStrokeStyle(3);
+    underLine.graphics.beginStroke('#FF4500');
+    underLine.graphics.moveTo(0, height + 2);
+    underLine.graphics.lineTo(width, height + 2);
+    underLine.graphics.endStroke();
+    this.addChild(underLine);
+    this.underline = underLine;
+    this.underline.visible = false;
```

Comment:
This is underlining the whole label.  The desired behavior for addressing this todo is to only underline the first instance of a letter that corresponds to the acessKey.

Reply:
updated